etc/oracle.yaml

```diff
@@ -15,11 +15,16 @@ blockchain:
   #endpoint: https://rinkeby.infura.io/00iTrs5PIy0uGODwcsrb
   #sidechain_endpoint: https://sidechain-dev.sonm.com
   #contract_registry: 0xaf1ffd7f652be7e9a0854a42b2d3046f853f80f1
+  blocks_batch_size: 50000
 
 oracle:
   # In master mode this instance submitting price changing
   # In slave mode this instance confirm changing
   is_master: false
+  #  from_now` parameter forces Oracle to process transactions only from the current block, otherwise, it will read whole transaction history from the beginning of times.
+  #  This parameter works only if slave mode is active.
```

Comment:
I don't spot any sanity check for this condition. What will happen if both `is_master` are `from_now` are set to true?

Reply:
in this case `from_now` will be useless.